dev_tools/conf/mypy.ini

```diff
@@ -0,0 +1,16 @@
+[mypy]
+
+[mypy-__main__]
+follow_imports = silent
+ignore_missing_imports = true
+
+# 3rd-party libs for which we don't have stubs
+[mypy-apiclient.*,numpy.*,pytest.*,scipy.*,,setuptools.*,_pytest.*]
+follow_imports = silent
+ignore_missing_imports = true
+
+#Adding "sympy.* or mypy-sympy to the above list (3rd-party libs for which we don't have stubs) doesn't ignore "cannot find module 'sympy' error
```

Comment:
nit: space after # for consistency with comment above

Reply:
Done